docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,428 @@
+For example, if the cluster consumer has cluster ID `cl-consumer`, its tenant namespace might look like the following:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    liqo.io/remote-cluster-id: cl-consumer
+    liqo.io/tenant-namespace: "true"
+  name: liqo-tenant-cl-consumer
+spec: {}
+```
+
+## Declarative network configuration
+
+By default, the network connection between clusters is established using a secure channel created via [Wireguard](https://www.wireguard.com/).
+To allow the creation of the tunnel, one cluster, typically the provider, needs to host a server gateway that exposes a port for the client gateway (usually running on the consumer cluster) to connect to.
+
+Therefore, it is required that the server gateway is reachable from the peer cluster, to enable the creation of the tunnel between the two clusters.
+
+In this guide, **we will configure the client gateway on the consumer cluster and the server gateway on the provider cluster**, which is the most common setup.
+However, depending on your requirements, you may choose to reverse these roles.
+
+### Creating and exchanging the network configurations (both clusters)
+
+The clusters that needs to be connected requires the network configuration of the peer cluster, which is provided via the `Configuration` CR.
+You can check an example of the resources to apply at the [following documentation page](./inter-cluster-network.md#definition-of-the-network-configuration-configuration-crds).
+
+**The Configuration resource should be applied in both clusters** and it **must contain the pod and external CIDR of the peer cluster**. (cluster A has the CIDR config of B and vice versa).
+
+The CIDRs are defined at installation time, you can check the values of the pod and the external CIDR configured in the local cluster by running:
+
+```{code} bash
+kubectl get networks.ipam.liqo.io -n liqo
+
+NAME            DESIRED CIDR    REMAPPED CIDR   AGE
+external-cidr   10.70.0.0/16    10.70.0.0/16    3h16m
+internal-cidr   10.80.0.0/16    10.80.0.0/16    3h16m
+pod-cidr        10.244.0.0/16   10.244.0.0/16   3h16m
+service-cidr    10.96.0.0/16    10.96.0.0/16    3h16m
+```
+
+### Creating and exchanging the Wireguard keys (both clusters)
+
+To enable authentication and encryption, Wireguard requires a key pair, one for each gateway.
+
+Those keys can be created [via the Wireguard utility tool](https://www.wireguard.com/quickstart/#key-generation) or via OpenSSL like so:
+
+```{code} bash
+openssl genpkey -algorithm X25519 -outform der -out private.der
+openssl pkey -inform der -in private.der -pubout -outform der -out public.der
+
+# Get the Wireguard private key
+echo "Private key:"; cat private.der | tail -c 32 | base64
+# Get the Wireguard public key
+echo "Public key:"; cat public.der | tail -c 32 | base64
+```
+
+At this point, **in each cluster**, we will need to create a secret containing this pair of keys:
```

Comment:
```suggestion
At this point, you need to create **in each cluster** a secret containing this pair of keys:
```

Reply:
Be uniform. If you use "we", we use "we" always. In any case, "you" is usually preferred to "we", and often we avoid using the future (hence "you need to" instead of "we will need to") to be more concise.